app/pages/_app.tsx

```diff
@@ -27,6 +32,24 @@ function MyApp({ Component, pageProps }: AppProps) {
     setError(null);
   }, [Component]);
 
+  useMemo(() => {
+    const fetchGrowthbook = async () => {
+      try {
+        const data = await fetch(
+          `https://cdn.growthbook.io/api/features/${publicRuntimeConfig.GROWTHBOOK_API_KEY}`
+        )
+          .then((res) => res.json())
+          .then((res) => {
+            growthbook.setFeatures(res.features);
+          });
```

Comment:
this is not a response object anymore
```suggestion
          .then((jsonData) => {
            growthbook.setFeatures(jsonData.features);
          });
```